compiler/rustc_mir_dataflow/src/elaborate_drops.rs

```diff
@@ -889,12 +866,7 @@ where
                 }
             }
             ty::Dynamic(..) => self.complete_drop(self.succ, self.unwind),
-            ty::Array(ety, size) => {
-                let size = size.try_eval_target_usize(self.tcx(), self.elaborator.param_env());
-                self.open_drop_for_array(*ety, size)
-            }
-            ty::Slice(ety) => self.open_drop_for_array(*ety, None),
-
+            ty::Array(ety, _) | ty::Slice(ety) => self.open_drop_for_array(*ety, None),
```

Comment:
nit, don't apply yet since bors is `try`ing: we can probably just remove this `None` argument, unless there are other callers to `open_drop_for_array`?